example/docker-helper/docker_helper/helper.py

```diff
@@ -9,6 +10,12 @@
 
 class DockerHelper(object):
     def setup(self):
+        # Filter out unclosed socket ResourceWarnings. These happen due to
+        # requests' connection pooling and spam the logs unnecessarily:
+        # https://github.com/requests/requests/issues/1882
```

Comment:
I assume there's no way to get at the `requests` stuff in the `docker` library so we can tell it not to reuse connections?